### PR TITLE
fixed bug in posts/index.html.erb to incorporated changes from multip…

### PR DESCRIPTION
**Pull request by SimonTanner**
…le branches

There was an issue with the aforementioned file which I've now resolved. There is however a weird issue with the comments and posts being displayed twice. I didn't want to change this as I hadn't worked on that part so I think this needs some attention from the groups that worked on the front-end styling and those who were working on the comments on a post. Otherwise it works and fixed 3 of the 5 failing rspec tests.

### CODE REVIEW

**Comment by FiddlersCode** [comment]
:shipit

**Comment by FiddlersCode** [comment]
don't know why the :squirrel isn't working! Github being really slow right now

**Comment by chrisjmit** [comment]
fixed conflicts

**Review by chrisjmit — APPROVED** [review]
:shipit:

**Comment by sblausten** [comment]
:shipit:

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<% @posts.each do |post| %>
 <% @post = post %>
```
Comment (sblausten) [inline app/views/posts/index.html.erb]: Is this necessary? Can't we just use the local post variable for the second nested #each?

**Review by sblausten — COMMENTED** [review]
Unnecessary instance variable used I think?

**Comment by dbugsy** [comment]
@SimonTanner this has been in QA for a while - would be nice to get this merged :)

**Pull request merged**